pkg/adaptor/cloud/azure/provider.go

```diff
@@ -213,16 +214,23 @@ func (p *azureProvider) CreateInstance(ctx context.Context, podName, sandboxID s
 		securityProfile = nil
 	}
 
+	imgRef := &armcompute.ImageReference{
+		ID: to.Ptr(p.serviceConfig.ImageId),
+	}
+	if strings.HasPrefix(p.serviceConfig.ImageId, "/CommunityGalleries/") {
```

Comment:
For my understand, is this the doc for community gallery - https://learn.microsoft.com/en-us/azure/virtual-machines/share-gallery-community?tabs=cli ?

Reply:
Thanks @jepio. I'll try the community gallery. And also get back on the community RHEL image.